cpu/stm32/periph/ptp.c

```diff
@@ -61,10 +61,10 @@
 #endif /* !STM32_PTPSSIR */
 
 /**
- * @brief   Return the result of x / y, scientifically rounded
+ * @brief   Return the result of x / y, rounded
  * @param   x       Number to divide
  * @param   y       @p x should be divided by this
- * @return  x/y, scientifically rounded
+ * @return  x/y, rounded to the nearest integer
```

Comment:
```suggestion
 * @return  x/y, rounded to the nearest integer. Edge cases are rounded towards infinitiy
```

Reply:
while i certenly like to talk about infinity and beyond i don't think this is the easy way to describe this, but it wil take this as i need to be more specific.